from vonage_utils.errors import VonageError


class UsersError(VonageError):
    """Indicates an error with the Vonage Users Package."""
